buildkite/src/Jobs/Release/MinaArtifactBullseyeDevnetLegacy.dhall

```diff
@@ -0,0 +1,26 @@
+let ArtifactPipelines = ../../Command/MinaArtifact.dhall
+
+let Artifacts = ../../Constants/Artifacts.dhall
+
+let Pipeline = ../../Pipeline/Dsl.dhall
+
+let PipelineTag = ../../Pipeline/Tag.dhall
+
+let Network = ../../Constants/Network.dhall
+
+let DebianVersions = ../../Constants/DebianVersions.dhall
+
+in  Pipeline.build
+      ( ArtifactPipelines.onlyDebianPipeline
+          ArtifactPipelines.MinaBuildSpec::{
+          , artifacts = [ Artifacts.Type.Daemon ]
+          , tags =
+            [ PipelineTag.Type.Long
+            , PipelineTag.Type.Release
+            , PipelineTag.Type.Docker
+            ]
+          , network = Network.Type.DevnetLegacy
+          , debVersion = DebianVersions.DebVersion.Bullseye
+          , if = Some "build.branch == 'dkijania/build_legacy_debian_on_comp'"
```

Comment:
Why do we need this condition?

Reply:
I want to change it to build.branch == 'compatible' after review is passed. My idea was to prevent building legacy package on different branch than compatible. However, we can allow it on custom branches and just to caution which package we are promoting